lib/runtime-core/src/backing.rs

```diff
@@ -605,8 +626,8 @@ fn import_functions(
             None => {
                 if imports.allow_missing_functions {
                     functions.push(vm::ImportedFunc {
-                        func: ::std::ptr::null(),
-                        vmctx: ::std::ptr::null_mut(),
+                        func: ptr::null(),
```

Comment:
Do we still need `func` here? (since we have `func_ctx` that perhaps already have it?)

Reply:
No worries, thanks for the explanation! :)